src/app/components/Book/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+
+import BookCover from '@assets/img_book1.png';
+
+import styles from './styles';
+
+function Book() {
+  return (
+    <View style={styles.bookContainer}>
+      <Image style={styles.bookCover} source={BookCover} />
+      <View style={styles.bookInfo}>
+        <Text style={styles.bookTitle}>A little Bird Told Me</Text>
+        <Text style={styles.bookAuthor}>Timothy Cross</Text>
```

Comment:
you could receive the image, title and author as props and then you can render different books with the same component

Reply:
On it! 🚀